src/app/events/event-registration-option/event-registration-option.component.html

```
@@ -46,8 +46,16 @@ <h3 class="title-medium">{{ registrationOption().title }}</h3>
             <button
               (click)="joinWaitlist(registrationOption())"
               mat-flat-button
+<<<<<<< HEAD
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)